CONTRIBUTING.md

```diff
@@ -2,29 +2,57 @@
 
 Operator Lifecycle Manager is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into operator-lifecycle-manager.
 
-# Email and Chat
+## Communication
 
-- Email: [operator-framework][operator_framework]  
+- Slack: [#olm-dev on the Kubernetes Slack](https://kubernetes.slack.com/archives/C0181L6JYQ2)
+- Google Group: [OLM Dev Google Group](https://groups.google.com/u/1/g/operator-framework-olm-dev)
```

Comment:
It looks like https://groups.google.com/g/operator-framework-olm-dev also works and doesn't contain (presumably) the user (`/u/1/...`) path encoded into the URL.